File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id            :integer          not null, primary key
#  flight_number :string
#  origin        :string
#  destination   :string
#  date          :datetime
#  airplane_id   :integer
#  created_at    :datetime         not null
#  updated_at    :datetime         not null
#

class Flight < ActiveRecord::Base
  belongs_to :airplane
  has_many :reservations
end

File: app/assets/javascripts/models/flight.js
var BurningAirlines = BurningAirlines || {};

// class BurningAirlines.Models.Flight extends Backbone.Model

BurningAirlines.Models.Flight = Backbone.Model.extend({
    url: function () {
      return '/flights/' + this.get('flight_id');
      console.log('url');
    },
    defaults: {
      flight_number: '',
      origin: '',
      destination: '',
      date: '',
      airplane_id: ''
    }
});

